e2e/jest-circus.setup.ts

```diff
@@ -2,4 +2,4 @@ require('expect-puppeteer');
 
 // Runs failed tests n-times until they pass or until the max number of retries is exhausted. This only works with jest-circus.
 // This file loaded in jest.config.js ==> setupFilesAfterEnv
-jest.retryTimes(process.env.DEBUG === 'true' ? 0 : 1);
+// jest.retryTimes(process.env.DEBUG === 'true' ? 0 : 1);
```

Comment:
disable global retry failed tests in an indiscriminate fashion. Retry uses up billing projects and may hides application issues.